docs/README.md

```diff
@@ -6,6 +6,12 @@ Scenarioo helps you visualizing and documenting the functionalities of your soft
 
 For more hi level introduction about what scenarioo is, please refer to our web page: http://www.scenarioo.org
 
+## Video: A Short Introduction to Scenarioo
+
+Shows the essential features of Scenarioo and gives you some first insights into the Scenarioo documentation format structure.
+
+<p><iframe width="883" height="497" src="https://www.youtube-nocookie.com/embed/C_Omyu190V4?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe></p>
```

Comment:
The documentation pages have a width of only 770 px. Please scale the video size down, so it's not cut off.

Reply:
ok, - will do it